<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Value</class>
 <widget class="QDialog" name="Value">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>340</y>
     <width>281</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Warehouse capacity</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>420</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>32</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Simulation data</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>100</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>340</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>180</y>
     <width>240</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Number of clients</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>220</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Number of days</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>430</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>341</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Number of product types</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>180</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>260</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Value</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>298</x>
     <y>443</y>
    </hint>
    <hint type="destinationlabel">
     <x>334</x>
     <y>400</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
